### PR TITLE
Flutter3.27

### PR DESCRIPTION
**Pull request by ericwindmill**
Updating samples for the next stable release

### CODE REVIEW

File: compass_app/app/lib/ui/home/widgets/home_screen.dart [hunk 1]
```diff
@@ -166,6 +167,18 @@ class _Booking extends StatelessWidget {
...
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.end,
+          children: [
+            Padding(
+              padding: const EdgeInsets.only(right: Dimens.paddingHorizontal),
+              child: Icon(Icons.delete),
+            ),
+          ],
+        ),
+      ),
```
Comment (ericwindmill) [inline compass_app/app/lib/ui/home/widgets/home_screen.dart:181]: This was just missing, it's not release related

File: material_3_demo/pubspec.yaml [hunk 2]
```diff
@@ -9,7 +9,6 @@ version: 1.0.0+1
 
 environment:
   sdk: ^3.5.0
-  flutter: ^3.19.0
```
Comment (ericwindmill) [inline material_3_demo/pubspec.yaml:12]: We shouldn't pin versions in the samples repo

Comment (domesticmouse) [reply material_3_demo/pubspec.yaml:12]: That's not a pin, but a minimum version. I'm guessing this sample relies on something that was introduced in that Flutter version.

Comment (ericwindmill) [reply material_3_demo/pubspec.yaml:12]: You're right, I guess I wasn't paying attention. One way or the other, the analyzer wasn't highlighting the warnings that I know were there. Removing this probably wasn't what fixed it, I'll revert

**Comment by domesticmouse** [comment]
I'd bump the minimum Dart/Flutter versions and re-run the `codelab_rebuild` scripts we have to rebuild the Runners. I'm happy to do that in a follow up PR if you'd prefer

**Review by domesticmouse — APPROVED** [review]

**Pull request merged**